.zuul.yaml

```diff
@@ -0,0 +1,30 @@
+---
+job:
+  - test-operator-podified-multinode-edpm-deployment-crc:
```

Comment:
this job does not exist yet

i'm not 100% sure, but I believe You have to 'define' it here (so instead of this line) like:
```
- job:
    name: test-operator-podified-multinode-edpm-deployment-crc
    parent: podified-multinode-edpm-deployment-crc
    dependencies: ["openstack-k8s-operators-content-provider"]
    vars:
        ...
    required-projects: ...
```